src/Resources/translations/admin.en.yml

```
@@ -5,6 +5,7 @@ plugin_pimcore_datahub_data_importer_configpanel_createLocationStrategy.type_adv
 plugin_pimcore_datahub_data_importer_configpanel_advanced_parent_path: Advanced Parent Path
 plugin_pimcore_datahub_data_importer_configpanel_locationUpdateStrategy.type_advancedParent: Advanced Parent
 plugin_pimcore_datahub_data_importer_configpanel_type_advancedXlsx: Advanced XLSX
+plugin_pimcore_datahub_data_importer_configpanel_type_optionalElementsXml: Optional Elements XML
```

```suggestion
plugin_pimcore_datahub_data_importer_configpanel_type_optionalElementsXml: XML Schema-based Preview
```
Do we make this change now? We may want to rename some more of the classes and files to suit?

We should also add a quick blurb to the readme docs.

Finally, Jeremy suggested we take 15 minutes to review if we could do a super() call to the preview loader then add the schema based fields instead of copy-pasting to replace it.